Project/backend/crytpo.py

```diff
@@ -5,87 +5,92 @@
 import json
 from dotenv import load_dotenv
 import os
+import time
 
 load_dotenv()
+starttime = time.time()
 
-reddit = praw.Reddit(
-    client_id = os.getenv("USER_ID"),
-    client_secret = os.getenv("CLIENT_SECRERT"),
-    user_agent = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36"
-)
-df = []
+while True:
+    reddit = praw.Reddit(
+        # client_id = os.getenv("USER_ID"),
+        # client_secret = os.getenv("CLIENT_SECRERT"),
+        client_id = "EUjAh6lhD_gNuQ",

```

Comment:
Make sure you aren't committing the API keys (It violates TOS for these APIs)

Reply:
Good call, should hopefully be fixed now.